gmail/Module.md

```diff
@@ -1,54 +1,30 @@
+## Overview
+Ballerina Gmail Connector provides the capability to send, read and delete emails through the Gmail REST API. It also provides the ability to read, trash, untrash and delete threads, ability to get the Gmail profile and mailbox history, etc. The connector handles OAuth 2.0 authentication.
 
-## Module Overview
+This module supports [Gmail API v1](https://developers.google.com/gmail/api).
 
-Ballerina Gmail Connector provides the capability to send, read and delete emails through the Gmail REST API. It also provides the ability to read, trash, untrash and delete threads, ability to get the Gmail profile and mailbox history, etc. The connector handles OAuth 2.0 authentication.
+## Prerequisites
 
-## Configurations
+Before using this connector in your Ballerina application, complete the following:
 
-Instantiate the connector by giving authentication details in the Gmail client config, which has built-in support for OAuth 2.0. Gmail uses OAuth 2.0 to authenticate and authorize requests. The Gmail connector can be minimally instantiated in the Gmail client config using the Access Token or by using the Client ID, Client Secret and Refresh Token.
+1. Create a [Google account](https://accounts.google.com/signup/v2/webcreateaccount?utm_source=ga-ob-search&utm_medium=google-account&flowName=GlifWebSignIn&flowEntry=SignUp). (If you already have one, you can use that.)
 
-**Obtaining Tokens to Run the Sample**
+2. Obtain tokens - Follow [this guide](https://developers.google.com/identity/protocols/oauth2)
 
-1. Visit [Google API Console](https://console.developers.google.com), click **Create Project**, and follow the wizard to create a new project.
-2. Go to **Credentials -> OAuth Consent Screen**, enter a product name to be shown to users, and click **Save**.
-3. On the **Credentials** tab, click **Create Credentials** and select **OAuth Client ID**.
-4. Select an application type, enter a name for the application, and specify a redirect URI (enter https://developers.google.com/oauthplayground if you want to use
-[OAuth 2.0 Playground](https://developers.google.com/oauthplayground) to receive the Authorization Code and obtain the
-Access Token and Refresh Token).
-5. Click **Create**. Your Client ID and Client Secret will appear.
-6. In a separate browser window or tab, visit [OAuth 2.0 Playground](https://developers.google.com/oauthplayground). Click on the `OAuth 2.0 Configuration`
- icon in the top right corner and click on `Use your own OAuth credentials` and provide your `OAuth Client ID` and `OAuth Client Secret`.
-7. Select the required Gmail API scopes from the list of API's, and then click **Authorize APIs**.
-8. When you receive your authorization code, click **Exchange authorization code for tokens** to obtain the refresh token and access token.
+## Quickstart
 
-You can now enter the credentials in the Gmail client config.
+To use the `Gmail` connector in your Ballerina application, update the .bal file as follows:
```

Comment:
Let's remove `` from the word Gmail

Reply:
Fixed by ddd90073069938a3660747b110c4643c53e57b4f